UKHomeOffice/nmsw-ui
# Test nmsw 1110

Test added to cover
NMSW 1110: Fix for External Fal1 allows both arrival/departure port to be same 
NMSW-1180: Showing error messages for FAL5 and 6 when user enters value with spl characters based on BE deleting or letting the user knows to change
NMSW 1115 - Fix bug throwing error for whitespace


## To test
```
npm run cypress:open:staging 
```
 select 
fal1-error-validation.feature
task-details-page.feature
or run in cl
```
npm run cypress:run:staging -- --spec cypress/e2e/features/check-your-answers-page.feature
npm run cypress:run:staging -- --spec cypress/e2e/features/passenger-status-check.feature
```
